handbook/product/README.md

```diff
@@ -334,11 +335,11 @@ To maximize your chances of having a feature accepted, requesters can visit the
 ### How feature requests are evaluated
 Digestion of these new product ideas (requests) happens at the **🎁🗣 Feature Fest** meeting.
 
-At the **🎁🗣 Feature Fest** meeting, the DRI (Head of Product) weighs all requests in the agenda. When the team weighs a request, it is immediately prioritized or put to the side.
+At the **🎁🗣 Feature Fest** meeting, the DRI (Head of Product) weighs all requests on the board. When the team weighs a request, it is immediately prioritized or put to the side.
```

Comment:
Link to board somewhere

Reply:
@zhumo Luke and I have agreed to meet to set up the board once this PR has merged. I'll update the handbook with the link then.